stellargraph/core/convert.py

```diff
@@ -150,3 +154,144 @@ def convert_edges(
     assert all(features is None for features in edge_features.values())
 
     return EdgeData(edges, node_data)
+
+
+NodeInfo = namedtuple("NodeInfo", ["ids", "features"])
```

Comment:
I'm a bit wary of generalising this too much, because the use case here is somewhat specialised (e.g. if `features_in_node = isinstance(node_features, str)` is false, the `features` list is empty). I can give it a more specific name, though. E.g.

```suggestion
SingleTypeNodeIdsAndFeatures = namedtuple("SingleTypeNodeIdsAndFeatures", ["ids", "features"])
```

Thoughts?

Reply:
I see what you mean. I'm happy with the renaming 👍